internal/common/common_test.go

```diff
@@ -145,6 +146,24 @@ func TestParseReferenceValues(t *testing.T) {
 	testutil.AssertEq(t, "builder image digests[0]", referenceValues.BuilderImageSHA256Digests[0], "9e2ba52487d945504d250de186cb4fe2e3ba023ed2921dd6ac8b97ed43e76af9")
 }
 
+func TestFromProvenance(t *testing.T) {
```

Comment:
Should we add `TestFromProvenance_Amber`?

Reply:
Done.

Reply:
Actually, renamed this one to `TestFromProvenance_Amber`. We can in the future add `TestFromProvenance_Slsav02`.

Reply:
Renaming is what I meant :)

Reply:
Added a (currently) negative test too, which uses a SLSA v1 provenance.